tests/components/homekit/test_type_locks.py

```diff
@@ -1,6 +1,8 @@
 """Test different accessory types: Locks."""
 import pytest
 
+from homeassistant.components.homekit.const import (
+    ATTR_VALUE, ATTR_DISPLAY_NAME)
```

Comment:
Since you only need `ATTR_VALUE`, one line is completely enough here:
`from homeassistant.components.homekit.const import ATTR_VALUE`